utils/telldus-core/Makefile

```diff
@@ -0,0 +1,68 @@
+# 
+# Copyright (C) 2010-2019 Telldus Technologies AB
+#
+# This is free software, licensed under the GNU General Public License v2.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=telldus-core
+PKG_VERSION:=2.1.2
+PKG_RELEASE:=1
+
+PKG_LICENSE:=LGPL-2.1
+PKG_BUILD_DEPENDS:=argp-standalone 
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://download.telldus.com/TellStick/Software/telldus-core/
+PKG_HASH:=a20f6c74814afc23312d2c93ebbb37fdea9deaaee05ae7b6a6275e11e4662014
+PKG_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/nls.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/telldus-core
+	SECTION:=utils
+	CATEGORY:=Utilities
+	TITLE:=Telldus TellStick USB interface
+	URL:=http://www.telldus.com
+	MAINTAINER:=Peter Liedholm <PeterFromSwe884@gmail.com>
+	DEPENDS:=+confuse +libftdi +libstdcpp $(ICONV_DEPENDS)
+    # +libstdcpp required as of now (could it be fixed?)
```

Comment:
Remove this

Reply:
> Remove this
@diizzyy 
Do you mean the commented line about libstdcpp?

Reply:
Yes

Reply:
@diizzyy 
Ok. Removed.